dev/org.eclipse.codewind.ui/src/org/eclipse/codewind/ui/internal/messages/messages.properties

```diff
@@ -388,6 +393,10 @@ AppOverviewEditorOpenSettingsErrorMsg=An error occurred while opening the projec
 AppOverviewEditorOpenSettingsNotFound=The project settings file ''.cw-settings'' could not be found or was not accessible.
 AppOverviewEditorNoConnection=Connection not available
 AppOverviewEditorNoApplication=Application not available
+AppOverviewEditorInjectMetricsEntry=Auto Metrics Injection
+AppOverviewEditorInjectMetricsOn=On
+AppOverviewEditorInjectMetricsOff=Off
+AppOverviewEditorInjectMetricsUnavailable=Unavailable for this language
```

Comment:
Is it really not available because of the language.  Does that mean that an appsody node project should support injecting metrics?

Reply:
It is language due to the implementation but you're right there is a bug which stops it working for Appsody at the moment.

Reply:
Will need to discuss with @tobespc but this follows the implementation in https://github.com/eclipse/codewind-vscode/pull/319/files#diff-6d44bd13ea616b343738e344ff796b94R142